canary/apps/angular/angularcli/src/styles.css

```diff
@@ -1,3 +1 @@
 /* You can add global styles to this file, and also import other style files */
-
-@import '~@aws-amplify/ui-angular/theme.css';
```

Comment:
Do we need this file still?

Reply:
It could go either way, but since this is a default autogenerated file, I’m fine with leaving it as is.